homeassistant/components/advantage_air/sensor.py

```diff
@@ -62,7 +65,7 @@ class AdvantageAirTimeTo(AdvantageAirAcEntity, SensorEntity):
     _attr_native_unit_of_measurement = ADVANTAGE_AIR_SET_COUNTDOWN_UNIT
     _attr_entity_category = EntityCategory.DIAGNOSTIC
 
-    def __init__(self, instance, ac_key, action):
+    def __init__(self, instance, ac_key, action) -> None:
```

Comment:
Do you know the type of ac_key and action here?

Reply:
Fixed